packages/integration-tests/tests/bridge_erc20.ts

```diff
@@ -84,11 +84,17 @@ describe('Bridging ERC20 tokens', async function () {
 
     // mint tokens as starting balances
 
-    await eth_testToken.mint(await env.eth.deployer.getAddress(), 10_000);
+    await eth_testToken
+      .mint(await env.eth.deployer.getAddress(), 10_000)
```

Comment:
nit we can add the num in a var

Reply:
I think that the future token rate limit PR will be a better place to do this, will keep an eye on it